### PR TITLE
[14.0] [FIX] fieldservice_google_map

### PR DESCRIPTION
**Pull request by brian10048**
Add view record to window action

### CODE REVIEW

**Comment by brian10048** [comment]
Before - missing Google Map in view
Screenshot from 2022-11-02 16-55-17

After - Google Map view available
Screenshot from 2022-11-02 16-55-31

**Review by max3903 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

File: fieldservice_google_map/views/fsm_order.xml [hunk 1]
```diff
@@ -70,4 +70,11 @@
         </field>
     </record>
 
+    <record id="action_orders_view_map" model="ir.actions.act_window.view">
+        <field name="sequence" eval="1" />
```
Comment (brian10048) [inline fieldservice_google_map/views/fsm_order.xml]:
```suggestion
        <field name="sequence" eval="2" />
```
@max3903 Maybe Kanban view should be first, not the map?

File: fieldservice_google_map/views/fsm_location.xml [hunk 2]
```diff
@@ -71,4 +71,14 @@
         <field name="view_mode">tree,form,google_map</field>
     </record>
 
+    <record id="action_locations_view_map" model="ir.actions.act_window.view">
+        <field name="sequence" eval="1" />
```
Comment (brian10048) [inline fieldservice_google_map/views/fsm_location.xml]:
```suggestion
        <field name="sequence" eval="2" />
```

File: fieldservice_google_map/views/fsm_order.xml (hunk 1 above)
Comment (max3903) [reply fieldservice_google_map/views/fsm_order.xml]: I agree 👍

**Comment by brian10048** [comment]
Screenshot from 2022-11-03 10-23-40

**Review by ghost — APPROVED** [review]

**Review by max3903 — APPROVED** [review]

**Pull request merged**